arm/Microsoft.Authorization/roleDefinitions/.bicep/nested_roleDefinitions_mg.bicep

```diff
@@ -1,34 +1,28 @@
 targetScope = 'managementGroup'
 
 param roleName string
-param roleDescription string = ''
+param description string = ''
```

Comment:
I'd suggest adding @description for param and output also for nested modules

Reply:
Fixed